projects/ngrx.io/content/guide/store/reducers.md

```diff
@@ -64,26 +65,42 @@ The initial values for the `home` and `away` properties of the state are 0.
 
 ### Creating the reducer function
 
-The reducer function's responsibility is to handle the state transitions in an immutable way. Define a reducer function that handles the actions for managing the state of the scoreboard.
+The reducer function's responsibility is to handle the state transitions in an immutable way. Create a reducer function that handles the actions for managing the state of the scoreboard using the `createReducer` function.
 
 <code-example header="scoreboard.reducer.ts">
-export const reducer = createReducer(
-  initialScoreState,
+const scoreboardReducer = createReducer(
+  initialState,
   on(ScoreboardPageActions.homeScore, state => ({ ...state, home: state.home + 1 })),
   on(ScoreboardPageActions.awayScore, state => ({ ...state, away: state.away + 1 })),
   on(ScoreboardPageActions.resetScore, state => ({ home: 0, away: 0 }))
 );
+
+export function reducer(state: State | undefined: action: Action) {
+  return scoreboardReducer(state, action);
+}
```

Comment:
This will only work for `StoreModule.forFeature` or for `ReducerActionMap`. I think we should recommend `combineReducers` way if there are more than a single reducer.

Reply:
Agreed. I think that should be an additional section to talk about `combineReducers`

Reply:
Haven't found any. Is it somewhere I missed? :)

Reply:
@maxime1992 there's an open issue - #1179